src/sc/embedder/content/embedder.py

```diff
@@ -216,19 +217,27 @@ def get_fallback(self, url):
 
     def load_oembed(self, action):
         url = self.widgets['url'].value
-        if url != '':
-            consumer = component.getUtility(IConsumer)
-            json_data = consumer.get_data(url, maxwidth=None, maxheight=None,
-                                          format='json')
+        if url == '':
+            return
+
+        v = validation.validatorFor('isURL')
+        if v(url) != 1:
+            plone_utils = api.portal.get_tool(name='plone_utils')
+            plone_utils.addPortalMessage(_(u'URL invalid'), 'info')
```

Comment:
you can use just ``api.portal.show_message()``

see: http://docs.plone.org/external/plone.api/docs/portal.html#portal-show-message-example